validation/src/test/scala/hmda/validation/rules/lar/validity/V570Spec.scala

```diff
@@ -0,0 +1,47 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.validation.dsl.Success
+
+class V570Spec extends RateSpreadEditCheckSpec {
+
+  override val check = V570
+
+  property("succeeds if lien status is 1 and rate spread is NA") {
+    succeedsWhen(lienStatus = 1, rateSpread = "NA")
+  }
+
+  property("fails if lien status = 1 and rate spread is < 1.5%") {
```

Comment:
Should there be an equivalent test to show that the check fails if the rate spread is `"100"` or greater?